script.js

```diff
@@ -0,0 +1,16 @@
+const crossbtn = document.querySelector('.cross');
+const menuMob = document.querySelector('.Items-display');
+const sanw = document.querySelector('.only-mob');
+const myname = document.querySelector('.name');
+
+crossbtn.addEventListener('click', () => {
+  menuMob.classList.remove('Menu-mobile-js');
+  myname.classList.remove('Menu-mobile-after');
+  sanw.classList.remove('Menu-mobile-after');
+});
+
+sanw.addEventListener('click', () => {
+  menuMob.classList.add('Menu-mobile-js');
+  myname.classList.add('Menu-mobile-after');
+  sanw.classList.add('Menu-mobile-after');
+});
```

Comment:
Closing the menu is not working as i click on the icon but the menu is not closing